.maintain/gitlab/check_polkadot_companion_build.sh

```diff
@@ -42,12 +42,16 @@ EOT
 
 SUBSTRATE_PATH=$(pwd)
 
+# Merge master into our branch before building Polkadot to make sure we don't miss
+# any commits that are required by Polkadot.
+git merge origin/master
```

Comment:
If we're going to do this for substrate should we also do it below for the polkadot branch?